include/pmacc/particles/ParticlesBase.hpp

```diff
@@ -172,6 +172,11 @@ class ParticlesBase : public SimulationFieldHelper<T_MappingDesc>
     /** copy gard particles to intermediate exchange buffer
      *
      * Copy all particles from the guard of a direction to the device exchange buffer.
+     *  @warning This method resets the number of particles in the processed supercells even
+     * if there are particles left in the supercell and not fulfill the rule that the
```

Comment:
At the very least there should be 'does' before `not`. Maybe the whole second part could be rewritten as `...and does not guarantee the last frame is ...` for clarity.